build_tools/benchmarks/post_benchmarks_as_pr_comment.py

```diff
@@ -274,6 +266,16 @@ def categorize_benchmarks_into_tables(benchmarks: Dict[
       raw[name] = results
       continue
 
+    similar_threshold = DEFAULT_SIMILAR_BECNHMARK_THRESHOLD
+    # Set different threshold for noisy benchmarks.
+    for regex, threshold in NOISY_BENCHMARKS:
+      if regex.match(name):
+        if threshold.endswith("%"):
+          similar_threshold = float(threshold.rstrip("%")) / 100
```

Comment:
This is exactly why I suggested passing the value through as a number instead of a string :P

```
#   There are two formats supported: a percentage or an absolute value. So we
#   use a string here.
```

only a percentage is used right now, so YAGNI - just pass it as a number and do the string formatting when actually handing it off for display.

Reply:
Probably. I still see the value of supporting both, especially for models that now finish within ~10ms. Using percentage is inconvenient for them---we will see a large percentage swing that is a bit scary but actually it just varies by 1ms or something. Using absolute numbers will be more comforting. (We can adjust the units but that's harder given old data wasn't with the same unit. Need to adjust everything.)

But yeah, given it's not there, I'm fine changing this. ;-P Done.

Reply:
That's fine, but there are better (more structured) ways to encode that than a string with or without a `%` character, like a dictionary with `threshold_percentage` or `threshold_absolute_ms` or a class with accessor methods.